app/models/log.rb

```diff
@@ -315,9 +315,11 @@ def plural_gender_for_person(person_num)
   end
 
   def update_status!
-    return if skip_update_status
-
-    self.status = calculate_status
+    if status == "pending"
+      self.status_cache = calculate_status
+    else
+      self.status = calculate_status
+    end
```

Comment:
I can't see why we would ever not want to recalculate status_cache when updating a pending log, but let me know if there is such scenario